### ISSUE TITLE
abinet,use_lang配置为False，报错

### ISSUE THREAD

**Issue by printfxs** [PaddlePaddle/PaddleOCR#9215]
2.6版本，abinet,use_lang配置为true时，正常训练，配置为False时，报错：
logits_dims[i]:832 != labels_dims[i]:256

**Comment by an1018** [comment]
具体可以参考下代码：https://github.com/PaddlePaddle/PaddleOCR/blob/366ad29d6c202a79bad103c72556c1186915c9c8/ppocr/modeling/heads/rec_abinet_head.py#L206，包含Linear层logits_dim会和label_dim一致，所以设置为True训练即可

**Comment by printfxs** [comment]
你好，我这边定位并不是linear层的问题，问题在rec_ce_loss中，并未为abinet的单独vision块预留处理接口，
我的修改如下：
在rec_abinet_head.py中，return logits 修改为 return {‘vision’： logits}
在rec_postprocess.py中，在 preds=preds['align'][-1]处，嵌套如下代码：
if 'align' in preds.keys():
     preds=preds['align'][-1.numpy()
else:
    preds=preds['vision']

**Comment by printfxs** [comment]
你好，我这边定位并不是linear层的问题，问题在rec_ce_loss中，并未为abinet的单独vision块预留处理接口，
我的修改如下：
在rec_abinet_head.py中，return logits 修改为 return {‘vision’： logits}
在rec_postprocess.py中，在 preds=preds['align'][-1]处，嵌套如下代码：
if 'align' in preds.keys():
     preds=preds['align'][-1.numpy()
else:
    preds=preds['vision']

**Comment by an1018** [comment]
收到，感谢反馈
